rb/lib/selenium/webdriver/remote/http/default.rb

```diff
@@ -28,6 +28,27 @@ module Http
         class Default < Common
           attr_accessor :proxy
 
+          attr_accessor :open_timeout
+          attr_accessor :read_timeout
+
+          # Initializes object.
+          # Warning: Setting {#open_timeout} to non-nil values will cause a separate thread to spawn.
+          # Debuggers that freeze the process will not be able to evaluate any operations if that happens.
+          # @param [Numeric] open_timeout - Open timeout to apply to HTTP client.
+          # @param [Numeric] read_timeout - Read timeout (seconds) to apply to HTTP client.
+          def initialize(open_timeout: nil, read_timeout: nil)
+            @open_timeout = open_timeout
+            @read_timeout = read_timeout
+          end
+
+          # Maintaining backward compatibility.
+          # @param [Numeric] value - Timeout in seconds to apply to both open timeout and read timeouts.
+          # @deprecated Please set the specific desired timeout {#read_timeout} or {#open_timeout} directly.
```

Comment:
I don't know when the accessor would be removed, so I've opted to use `Kernel#warn` instead.

Along the same logic, I think we should also deprecate `Common#timeout` and `Common#timeout=` and push that logic down to `Curb` since it's the only subclass that's using it. Otherwise we'll have an abstract class that has an otherwise valid property that's been deprecated (eventually removed) on one of its concrete implementations.